python/cuml/test/test_pickle.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import os
+import cuml
+from cuml.test.utils import array_equal, np_to_cudf
+import cudf
+import numpy as np
+import pandas as pd
+from sklearn.datasets import make_regression
+import pickle
+
+regression_models = dict(
+    LinearRegression=cuml.LinearRegression(),
+    Lasso=cuml.Lasso(),
+    Ridge=cuml.Ridge(),
+    ElasticNet=cuml.ElasticNet()
+)
+
+solver_models = dict(
+    CD=cuml.CD(),
+    SGD=cuml.SGD(eta0=0.005)
+)
+
+cluster_models = dict(
+    KMeans=cuml.KMeans()
+)
+
+decomposition_models = dict(
+    PCA=cuml.PCA(),
+    TruncatedSVD=cuml.TruncatedSVD(),
+    UMAP=cuml.UMAP()
+)
+
+neighbor_models = dict(
+    NearestNeighbors=cuml.NearestNeighbors()
+)
+
+
+def unit_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.unit)
+
+
+def quality_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.quality)
+
+
+def stress_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.stress)
+
+
+def make_cudf_series(arr):
+    df = pd.DataFrame(

```

Comment:
Not sure about the sequence here. Is arr a single-column array? Could you just do:
 `cudf.Series(arr)`